python/cugraph-service/server/pyproject.toml

```diff
@@ -3,6 +3,7 @@
 [build-system]
 
 requires = [
+    "numpy>=1.23,<3.0a0",
```

Comment:
`cugraph-service-server` should have a hard runtime dependency on `numpy`, because of this:

https://github.com/rapidsai/cugraph/blob/a6eea2f7cc40165e584e459351d1a92406de7415/python/cugraph-service/server/cugraph_service_server/cugraph_handler.py#L30

its conda recipe does:

https://github.com/rapidsai/cugraph/blob/a6eea2f7cc40165e584e459351d1a92406de7415/conda/recipes/cugraph-service/meta.yaml#L66

Reply:
agreed, thanks for being so thorough!